src/FormNetworks/quadraticformnetwork.jl

```diff
@@ -0,0 +1,86 @@
+default_index_map = prime
+
+struct QuadraticFormNetwork{V,FormNetwork<:BilinearFormNetwork{V},IndexMap} <:
+       AbstractFormNetwork{V}
+  formnetwork::FormNetwork
+  dual_index_map::IndexMap
+end
+
+bilinear_formnetwork(qf::QuadraticFormNetwork) = qf.formnetwork
+function QuadraticFormNetwork(
+  operator::AbstractITensorNetwork,
+  bra::AbstractITensorNetwork,
+  ket::AbstractITensorNetwork;
+  dual_index_map=default_index_map,
+  kwargs...,
+)
```

Comment:
Do we need this constructor? I was thinking we would just have `QuadraticFormNetwork(operator, ket)` and `QuadraticFormNetwork(ket)`.

Reply:
Good point it is redundant and a bad idea anyway